README.md

```diff
@@ -62,6 +62,31 @@ Follow linked crate name for detailed status. Please note that all crates follow
 
 [semver]: https://semver.org
 
+### Pipeline Integration
+
+Some CI/CD pipelines leverage repository cloning. Below is a copy-paste-able example to build docker images for such workflows.
+
+#### Pipeline Integration (recommended)
+
+Build an image without a target and then copy the binaries into your local image
+
+```dockerfile
+COPY --from gitoxide:latest /bin/gix /usr/local/bin/
+COPY --from gitoxide:latest /bin/ein /usr/local/bin/
+```
+
+#### Pipeline Integration (base image)
+
```

Comment:
I think there should be text here that explains what it's doing and why - it probably all flows together with the paragraphs above, and it feels like these could be re-ordered so the user first builds an image, then uses the image in some way, and of course also gets some idea on how to use it in a pipeline. My preference would be to show how to use such image with GitHub actions if at all possible.